stable/selenium/templates/hub-service.yaml

```diff
@@ -25,5 +25,8 @@ spec:
   - name: hub
     port: {{ .Values.hub.servicePort }}
     targetPort: {{ .Values.hub.port }}
+    {{- if .Values.hub.nodePort }}
```

Comment:
We should also check if `.Values.hub.serviceType` is `NodePort` before rendering this part of the service spec.

Reply:
I pushed the chart with this tests combination